LayoutTests/fast/css/all-keyword-direction.html

```diff
@@ -12,6 +12,10 @@
   direction:rtl;
   all: unset;
 }
+
+* {
+  unicode-bidi: normal;
+}
```

Comment:
What is this for? AFAICT it should have no effect on this file.

Reply:
It pushes `.` to front of text instead of end without this. I can either update test to retain old behavior with this change or can update `-expected.html` file and we match all other browsers (Chrome Canary 133 and Firefox Nightly 135) without this test change with new behavior. (JSFiddle - https://jsfiddle.net/j3gdqh0p/)

They also push `.` at front / start instead of end. So now it is up to us whether we update test or `-expected.html` file.

Reply:
Updated `-expected.htmo` file instead of test.